src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    chipVersion: version,
```

Comment:
```suggestion
    name,
    wheels,
    version,
```
Если принимаемое значение и поле объекта называются одинаково, то для краткости их можно записать так. принимаем